src/renderers/native/ReactNativeTagHandles.js

```diff
@@ -44,7 +44,7 @@ var ReactNativeTagHandles = {
   assertRootTag: function(tag: number): void {
     invariant(
       this.reactTagIsNativeTopRootID(tag),
-      'Expect a native root tag, instead got %s', tag
+      '\n\nExpect a native root tag, instead got %s', tag
```

Comment:
This doesn't look right to me. It puts a newline before the whole message, not before the addendum.
(There is no addendum here.)

Also, I would prefer if we avoided modifying `invariant` messages. Let’s stick to `warning`s for now.